src/main/java/uk/gov/companieshouse/api/accounts/validation/LocationType.java

```diff
@@ -5,6 +5,7 @@
  */
 public enum LocationType {
 
+    JSON_BODY("json-body"),
```

Comment:
I think `request-body` would be better here to keep the spec addition more generic - also, can you add to the spec so we don't forget about it :)